src/menu.ts

```diff
@@ -151,6 +151,18 @@ const applicationMenu: MenuItemConstructorOptions[] = [
           config.set(ConfigKey.LaunchMinimized, checked)
         }
       },
+      {
+        label: is.macos ? 'Hide Menu Bar Icon' : 'Hide System Tray Icon',
+        type: 'checkbox',
+        checked: config.get(ConfigKey.HideTrayIcon),
+        click({ checked }: { checked: boolean }) {
+          config.set(ConfigKey.HideTrayIcon, checked)
+          showRestartDialog(
+            checked,
+            is.macos ? 'hiding menu bar icon' : 'hiding system tray icon'
```

Comment:
This might read better by changing the setting to "Enable Menu Bar Icon" so then the dialog can say: "To disable the menu bar icon, please restart."

```suggestion
            is.macos ? 'the menu bar icon' : 'the system tray icon'
```

Reply:
OK, I'll make the code change. Thanks!